src/functionalTest/resources/scenarios/RIA-1485-record-granted-time-extension-application-in-wrong-state.json

```diff
@@ -0,0 +1,43 @@
+{
+  "description": "RIA-1485 record granted time extension application in wrong state",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "CaseOfficer",
+    "input": {
+      "eventId": "recordApplication",
+      "state": "prepareForHearing",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "applications": [],
+          "applicationDocuments": [{
+            "id": "1",
+            "value": {
+              "document_url": "{$FIXTURE_DOC1_PDF_URL}",
+              "document_binary_url": "{$FIXTURE_DOC1_PDF_URL_BINARY}",
+              "document_filename": "{$FIXTURE_DOC1_PDF_FILENAME}"
+            }
+          }],
+          "applicationSupplier": "The legal representative",
+          "applicationType": "Time extension",
+          "applicationReason": "some reason",
+          "applicationDate": "2019-01-01",
+          "applicationDecision": "Granted",
+          "applicationDecisionReason": "some decision reason"
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
```

Comment:
Is the '200' status code correct for scenarios in the wrong state?

Reply:
yes, it is always 200 (which is not good practice), but the response body is different. It contains error messages